src/main/java/Duke.java

```diff
@@ -86,29 +34,29 @@ public static void main(String[] args) {
                 continue;
             }
             if (nextInput[0].equals("list")) { //want to print out the task list
-                printList(taskList);
+                Task.printTaskList(taskList);
```

Comment:
consider creating a class to manage the inputs by the user rather than putting them under tasks. The purpose of the Task class is not to parse the input of the user